poker-app/components/Radio.js

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from 'react'
+import { Text, View } from 'react-native'
+import RadioForm, { RadioButton, RadioButtonInput, RadioButtonLabel } from 'react-native-simple-radio-button';
+
+let radio_props = [
+    { label: 'Big Blind', value: 0 },
+    { label: 'Small Blind', value: 1 },
+    { label: 'Dealer', value: 2 },
+    { label: 'D+1', value: 3 },
+    { label: 'D+2', value: 4 },
+    { label: 'D+3', value: 5 },
+    { label: 'D+4', value: 6 },
+    { label: 'D+5', value: 7 }
+]
+
+class Radio extends Component {
```

Comment:
Do you need both Radio and AutoRadio?

Reply:
No actually the Auto Radio is just a different way of implementing the same radio button component in a way with some more control what it can do, I want to make the button increment its position after any action and I put this in there to try out the different radio button options I can use